app/src/pages/inside/projectSettingsPageContainer/content/elements/fieldElement/fieldElement.scss

```diff
@@ -41,6 +41,17 @@
   font-size: 11px;
   line-height: 16px;
   font-family: $FONT-ROBOTO-REGULAR;
+
+  a {
```

Comment:
could it be handled in FormattedDesc component?

Reply:
Done.